### ISSUE TITLE
Temporarily provide a "Coronavirus" option in the disease preferences

### ISSUE THREAD

**Issue by AlwaysLivid** [FoldingAtHome/fah-control#47]
A lot of people, myself included, have recently downloaded (or re-downloaded) FoldingAtHome, due to the amassed attention that it has received in the middle of the COVID-19 global pandemic, which is currently ongoing as of the time of this writing.

It would be great if people could focus on the particular goal that was the primary motivator that got them interested in the project in the first place, with the addition of an additional option that would allow this.

I have also opened a similar issue for the web GUI here: https://github.com/FoldingAtHome/fah-web-client/issues/13

**Comment by smitty42** [comment]
So, I'm new here and wanted the same thing, but after poking around the repos a bit, it looks to me like this will require a backend change to the client and or server API. And I don't think either of those is open source. 

My guess is that this would probably be a nontrivial change requiring updates to the client and server backends along with the more trivial frontend changes. Might require some db shinanigans depending on how all that's set up.

I'm guessing whoever is working on those components has their hands full as it is and trust they are triaging issues appropriately. I'd be happy to help out though if that was possible.

**Comment by shorttack** [comment]
Already fixed. Please try the beta at [](https://foldingathome.org/beta/) and report back here if you are still having this difficulty.
_Issue Support_

**Comment by shorttack** [comment]
closed: worksforme

**Comment by AlwaysLivid** [comment]
Yeah, it was just that when I opened this issue, the problem still happened to be there. Thanks for the help nevertheless, @shorttack.

**Comment by FlorinAndrei** [comment]
@shorttack Can we close issues only when the stable version is released? That's the general procedure with most software projects.

**Comment by shorttack** [comment]
@FlorinAndrei Good point. Stable version has been released.
